internal/ast/precedence.go

```diff
@@ -181,6 +172,15 @@ const (
 	OperatorPrecedenceLowest        = OperatorPrecedenceComma
 	OperatorPrecedenceHighest       = OperatorPrecedenceParentheses
 	OperatorPrecedenceDisallowComma = OperatorPrecedenceYield
+	// ShortCircuitExpression:
+	//     LogicalORExpression
+	//     CoalesceExpression
+	// CoalesceExpression:
+	//     CoalesceExpressionHead `??` BitwiseORExpression
+	// CoalesceExpressionHead:
+	//     CoalesceExpression
+	//     BitwiseORExpression
+	OperatorPrecedenceCoalesce = OperatorPrecedenceLogicalOR
```

Comment:
Alternatively, I could make the `OperatorPrecedenceCoalesce` lower than `OperatorPrecedenceLogicalOR` and then handle that change from Strada in `parseBinaryExpressionRest` and its computation of `consumeCurrentOperator` (and maybe in some other places too). cc @weswigham

Reply:
Agh, my bad, read the other PR wrong and forgot to expand some code.